src/components/common/Footer.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import "./index.css";
+import { Form, Container, Row, Col, Button, Nav } from "react-bootstrap";
+const Footer = () => {
+  const [email, setEmail] = useState("");
+  const handleSubmit = (e) => {
+    e.preventDefault();
```

Comment:
write something like this, this is how we do it normally: `// TODO(mygithubname): connect this to newsletter`